covid_xprize/examples/prescriptors/neat/prescribe.py

```diff
@@ -22,6 +24,11 @@
 from covid_xprize.examples.prescriptors.neat.utils import IP_MAX_VALUES
 from covid_xprize.examples.prescriptors.neat.utils import PRED_CASES_COL
 
+# Path to where this script lives
+ROOT_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
+
+# Config file for running NEAT (expected to reside in same dir as this script)
+NEAT_CONFIG_FILE = ROOT_DIR / 'config-prescriptor'
```

Comment:
We now navigate to the config file relative to where this script resides without making assumptions about the current working dir.